routes/web.php

```diff
@@ -17,20 +17,6 @@
     return view('welcome');
 });
 
-Route::get('/save', [\App\Http\Controllers\HistoricalWeatherHumidityController::class, 'saveHumidityToDb']);
+Route::get('/save', [\App\Service\HistoricalHumidityProcessingService::class, 'saveHumidityProcessToDb']);
```

Comment:
you can "import" the HistoricalHumidityProcessingService, so only the class name is shown here

- [ ] use/import class (simplify fqcn in PhpStorm)